### ISSUE TITLE
Installing lerna still shows one deprecated warnings for used module functionality

### ISSUE THREAD

**Issue by mleenaerts** [lerna/lerna#3605]
When installing lerna via npm install still one warning is shown about the usage of deprecated functionalities of used modules. 
This behaviour is still present in release 6.6.0 although a previous PR should have corrected it, but it seems only one of the two reported issues is revolved. (See https://github.com/lerna/lerna/issues/3550 and https://github.com/lerna/lerna/pull/3559)

Still present is the following warning:
npm WARN deprecated @npmcli/move-file@2.0.1: This functionality has been moved to @npmcli/fs

Expected Behaviour
I would prefer the replacement of the deprecated code to avoid potential future issues as soon as possible.

Steps to Reproduce
Create an empty folder and create a package.json file in that folder with the following content.

```
{
  "name": "test-app",
  "devDependencies": {
    "lerna": "6.6.0"
  }
}
```
Run npm install and notice the warning message regarding deprecated functionality.

Failure Logs / Configuration
npm WARN deprecated @npmcli/move-file@2.0.1: This functionality has been moved to @npmcli/fs

**Comment by markmxr** [comment]
Experiencing the same issue over here and it's breaking our pipelines.
```
Run npm install -g lerna@5.6.1
npm WARN deprecated readdir-scoped-modules@1.1.0: This functionality has been moved to @npmcli/fs
npm WARN deprecated @npmcli/move-file@2.0.1: This functionality has been moved to @npmcli/fs
npm ERR! code ENOENT
npm ERR! syscall chmod
npm ERR! path /opt/hostedtoolcache/node/16.10.0/x6[4](https://github.com/ManageXR/mighty-api/pull/106/checks#step:4:5)/lib/node_modules/lerna/node_modules/@nrwl/tao/index
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, chmod '/opt/hostedtoolcache/node/1[6](https://github.com/ManageXR/mighty-api/pull/106/checks#step:4:7).[10](https://github.com/ManageXR/mighty-api/pull/106/checks#step:4:11).0/x64/lib/node_modules/lerna/node_modules/@nrwl/tao/index'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent
```

Looks like it could be caused by the latest update to @nrwl/tao

**Comment by markmxr** [comment]
This is happening on node v16.10.0 and not on v16.19.1

> Experiencing the same issue over here and it's breaking our pipelines.
> 
> ```
> Run npm install -g lerna@5.6.1
> npm WARN deprecated readdir-scoped-modules@1.1.0: This functionality has been moved to @npmcli/fs
> npm WARN deprecated @npmcli/move-file@2.0.1: This functionality has been moved to @npmcli/fs
> npm ERR! code ENOENT
> npm ERR! syscall chmod
> npm ERR! path /opt/hostedtoolcache/node/16.10.0/x6[4](https://github.com/ManageXR/mighty-api/pull/106/checks#step:4:5)/lib/node_modules/lerna/node_modules/@nrwl/tao/index
> npm ERR! errno -2
> npm ERR! enoent ENOENT: no such file or directory, chmod '/opt/hostedtoolcache/node/1[6](https://github.com/ManageXR/mighty-api/pull/106/checks#step:4:7).[10](https://github.com/ManageXR/mighty-api/pull/106/checks#step:4:11).0/x64/lib/node_modules/lerna/node_modules/@nrwl/tao/index'
> npm ERR! enoent This is related to npm not being able to find a file.
> npm ERR! enoent
> ```
> 
> Looks like it could be caused by the latest update to @nrwl/tao

**Comment by amorscher** [comment]
@mleenaerts as mentioned in #3559. The issue was only partially fixed. Some package requiring this deprecated dep is still not updated. So the issue should not have been closed. I will have another look into this. Maybe we could now update the evil package version.

**Comment by mleenaerts** [comment]
@amorscher I guess I trusted the comments and didn't check myself as well. But anyways, your support and feedback was and is much appreciated.

**Comment by JamesHenry** [comment]
Actually sadly I think it will still get dragged in by `@npmcli/run-script`:

```
❯ npm ls @npmcli/move-file
lerna-monorepo@
└─┬ @npmcli/run-script@6.0.0
  └─┬ node-gyp@9.0.0
    └─┬ make-fetch-happen@10.2.0
      └─┬ cacache@16.1.1
        └── @npmcli/move-file@2.0.0
```

**Comment by JamesHenry** [comment]
@lukekarrys sorry for the direct ping but do you know if there are any near term plans to update this usage so that the deprecation warning can be avoided?

**Comment by amorscher** [comment]
I think it is due to node-gyp. Which currently has a pull request opened (https://github.com/nodejs/node-gyp/pull/2796) to move to a newer version of make-fetch-happen. But this is somehow blocked...

**Comment by lukekarrys** [comment]
I'm the author of a PR to `node-gyp` that would alleviate any deprecation warnings that are caused by it: https://github.com/nodejs/node-gyp/pull/2770

It would be a major bump for `node-gyp` but would then be released in a patch of `@npmcli/run-script`. That PR is currently awaiting review so it has fallen off in priority a bit as I wait for that.

**Comment by banyaner** [comment]
@markmxr  node v16.19.1 lerna 6.6.1 still has the issue

**Comment by Anutrix** [comment]
What's the status on this, now that https://github.com/nodejs/node-gyp/pull/2770 is merged?

**Comment by mleenaerts** [comment]
I still see the warning when installing lerna v8.0.0, if that's what you mean.
```npm WARN deprecated @npmcli/move-file@2.0.1: This functionality has been moved to @npmcli/fs```

**Comment by JamesHenry** [comment]
All the relevant npm internal deps have now received the relevant updates thanks to Luke, so I've merged #3919 which addresses the issue on the lerna side (verified locally).

Will cut a release later today which contains the update
